tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.robot

```diff
@@ -0,0 +1,166 @@
+# Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 5-22 - NFS Volume
+Resource  ../../resources/Util.robot
+Suite Setup  Setup NFS Server
+Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
+
+
+*** Variables ***
+${createFileContainer}=  createFileContainer
+${addToFileContainer}=  addToFileContainer
+
+
+*** Keywords ***
+Setup NFS Server
+    Log To Console  \nStarting test...
+    ${esx3}  ${esx4}  ${esx5}  ${vc}  ${esx3-ip}  ${esx4-ip}  ${esx5-ip}  ${vc-ip}=  Create a Simple VC Cluster  datacenter1  cls1
+
+    ${nfs_default_name}  ${nfs_default_ip}=  Deploy Nimbus NFS Datastore  %{NIMBUS_USER}  %{NIMBUS_PASSWORD}
+
+    Set Global Variable  @{list}  ${esx3}  ${esx4}  ${esx5}  ${vc}  ${nfs_default_name}
+
+    Set Suite Variable  ${NFS_DEFAULT_IP}  ${nfs_default_ip}
+    Set Suite Variable  ${NFS_DEFAULT_NAME}  ${nfs_default_name}
+
+    #Need to set nfs volume at vic creation time
+    Install VIC Appliance To Test Server  additional-args=--volume-store="nfs://${NFS_DEFAULT_IP}/store?uid=0&gid=0:nfsVolumeStoreDefault"
+
+Verify NFS Volume Basic Setup
+    [Arguments]  ${prevOutput}  ${ContainerName}  ${nfsIP}  ${rwORro}
+
+    ${rc}  ${outputTemp}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --name ${ContainerName} -d -v ${prevOutput}:/mydata ${busybox} mount
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} start -a ${ContainerName}
```

Comment:
So I was running `run` with `-d` which would cause STDOUT to not be written out.  @matthewavery told me what `-d` was doing in this case.  I tried the `run` command w/o `-d` and now I'm seeing the output I wanted.
I'm rewriting this to make it more simple now that I don't need two commands to get the output I needed.